dev/com.ibm.ws.jaxrs.2.1.client_fat/build.gradle

```diff
@@ -9,17 +9,17 @@
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
 repositories {
-  if (isUsingArtifactory) {
-    maven {
-      credentials {
-        username userProps.getProperty("artifactory.download.user")
-        password userProps.getProperty("artifactory.download.token")
-      }
-      url ("https://" + gradle.userProps.getProperty("artifactory.download.server") + "/artifactory/wasliberty-maven-remote")
-    }
-  } else {
-    mavenCentral()
-  }
+  if (isUsingArtifactory) { 
```

Comment:
Looks like this one could be updated to use 
`apply from: '../wlp-gradle/subprojects/maven-central-mirror.gradle'`

Reply:
Originally I had this one using the common gradle file, but the FAT started failing with a non-obvious issue and I didn't want to delay these changes any further. The only difference between the two is that the common file does `repositories.clear()` and the original file did not, so that must be where the behavior difference was